client/src/containers/Economy/components/Compare/components/Table/index.tsx

```diff
@@ -19,6 +20,9 @@ export const Table: FC<TableProps> = observer(({ indicator, unit }) => {
   return (
     <Fetch country={'all'} indicator={indicator}>
       {({ data, isLoading, error }) => {
+        const countriesAll =  JSON.parse(JSON.stringify(countries));
+        const dataFiltered = data.filter(obj => obj.countryiso3code.toLowerCase() in countriesAll);
```

Comment:
```suggestion
        const dataFiltered = data.filter(obj => countriesAll.includes(obj.countryiso3code.toLowerCase());
```

Reply:
@MaksymRudnyi There is difference between those two operators. Please check [stackoverflow](https://stackoverflow.com/questions/48499181/what-is-the-difference-between-in-operator-and-includes-for-javascript-arr#:~:text=includes()%20checks%20for%20the,in%20the%20case%20of%20arrays).)

So I am NOT looking for filtering item, but just keys.